hadoop-ozone/dist/src/main/compose/ozone-mr/hadoop32/.env

```diff
@@ -16,6 +16,6 @@
 
 HDDS_VERSION=@hdds.version@
 HADOOP_IMAGE=flokkr/hadoop
-HADOOP_VERSION=3.2.1
+HADOOP_VERSION=3.2.2
```

Comment:
@elek this change [needs](https://github.com/apache/ozone/pull/1795/checks?check_run_id=1699324055#step:7:442) a new `flokkr/hadoop:3.2.2` image, too.  I have [opened a PR](https://github.com/flokkr/docker-hadoop/pull/8) for that.  (Or we can keep MR acceptance tests on Hadoop 3.2.1.)

Reply:
Thanks, I merged the pr. Let's give it a few minutes and we can restart the builds.

Reply:
Thanks @elek.  There is still no new image built.  May be caused by the [problem with Hadoop 3.2.2 mirrors](https://mail-archives.apache.org/mod_mbox/hadoop-general/202101.mbox/%3C07913537-6683-414b-94c9-6846d0967db0%40www.fastmail.com%3E).